pkg/apis/networking/register.go

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package networking
+
+const (
+	GroupName = "networking.internal.knative.dev"
+
+	// ClassAnnotationKey is the annotation for the explicit class of
+	// ClusterIngress that a particular resource has opted into. For
+	// example,
+	//
+	//    networking.knative.dev/class: foo
+	//
+	// This uses a different domain because unlike the resource, it is
+	// user-facing.
+	ClassAnnotationKey = "networking.knative.dev/class"
```

Comment:
Can you give an example to tell how we will use this?

Reply:
Done.